### ISSUE TITLE
Polymorphic association generates migration with syntax error.

### ISSUE THREAD

**Issue by joshuapinter** [plentz/lol_dba#64]
Hey there. 

Using v 2.0.1 out of master. I have an Attachment model that is polymorphic with the following definition in the model: 

```ruby
class Attachment < ActiveRecord::Base
  belongs_to :attachable,  polymorphic: true
end
```

Using `rake db:find_indexes` it generates the following migration suggestion: 

```ruby
class AddMissingIndexes < ActiveRecord::Migration
  def change
    add_index :attachments, [:["attachable_id", "attachable_type"]]
  end
end
```

And when running `rake db:migrate` I get: 

```bash
syntax error, unexpected '[', expecting tSTRING_CONTENT or tSTRING_DBEG or tSTRING_DVAR or tSTRING_END
    add_index :attachments, [:["attachable_id", "attachable_type"]]
```

I can look into it if you want to point me in the right direction...

Thanks!

**Comment by Vorob-Astronaut** [comment]
have the same error too

**Comment by plentz** [comment]
@joshuapinter @Vorob-Astronaut hey guys. I've just released `2.0.2` with a fix for this. Can you test it, please? Feedback appreciated :)

**Comment by plentz** [comment]
(duplicates #62, btw)
